test/boulder-tools/build.sh

```diff
@@ -5,7 +5,7 @@ apt-get update
 
 apt-get install -y --no-install-recommends \
   libltdl-dev \
-  mariadb-client-core-10.0 \
+  mariadb-client-core-10.1 \
```

Comment:
Is this being bumped as a dep for softhsm2?

Reply:
`mariadb-client-core-10.0` is not provided by `stretch`.

Reply:
Doh! You mentioned that in the PR desc. Thanks!